Sources/Socket/SocketUtils.swift

```diff
@@ -152,7 +152,14 @@ public extension fd_set {
 
     @inline(__always)
     private static func address(for fd: Int32) -> (Int, Int32) {
-        let intOffset = Int(fd) / __fd_set_count
+        var intOffset = Int(fd) / __fd_set_count
+		#if arch(s390x)
```

Comment:
It would be better to use `#if _endian(big)`.

Reply:
Please also indent using spaces instead of tabs.

Reply:
This project uses _tabs_, so leave it as is.  However, the _#if _endian(big)_ should be made.  Thanks.

Reply:
Ah, the problem is that the rest of this function is indented with spaces not tabs, so it looked wrong.

Reply:
I'll correct that when I do the merge...